src/components/Global/MarkdownContent.vue

```diff
@@ -0,0 +1,25 @@
+<script setup>
+import { parse } from 'marked';
+import { computed, defineProps } from 'vue';
+
+const props = defineProps({
+  markdown: String,
+});
+
+const text = computed(() => parse(props.markdown, {
+  headerIds: false,
+  mangle: false,
+}));
+</script>
+
+<template>
+  <div class="markdown" v-html="text"></div>
+</template>
+
+<style lang="scss">
+.markdown {
+  p {
+    padding-bottom: 16px;
+  }
+}
```

Comment:
We can also target the tags insider markdown easily-- this fixes a spacing bug on the homepage. Eventually we should move these to `em`s though.

![image](https://github.com/cuttle-cards/cuttle/assets/414475/630463b0-ce52-43d4-b3a0-dff72d5fb7ec)